gtfs-realtime-validator-lib/pom.xml

```diff
@@ -7,11 +7,11 @@
     <parent>
         <groupId>edu.usf.cutr</groupId>
         <artifactId>gtfs-realtime-validator</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <version>${revision}</version>
```

Comment:
Should this be `${parent.version}` as well?

Seems like everything should be referencing the parent version consistently if it's outside root `pom.xml`.

Reply:
I believe it should be `${revision}`. I did tri `${parent.version}` at first, but I got an error mentioning that properties in `<parent>` are prohibited. Here is what we have right now:
<img width="724" alt="Capture d’écran, le 2022-02-16 à 15 39 43" src="https://user-images.githubusercontent.com/46797220/154353621-3f468a21-4b4c-4bde-9bcc-55cb9e776f76.png">
<img width="446" alt="Capture d’écran, le 2022-02-16 à 15 39 56" src="https://user-images.githubusercontent.com/46797220/154353624-9ca492a8-d90f-461e-9b2d-03d877a8955d.png">
So `${revision}` is passed from the command line to the root `pom.xml`, then to the modules `pom.xml`. Let me know if you still think that it is incorrect.